package gg.umbra.event.impl;

import gg.umbra.event.impl.EventRenderTickBase;
import gg.umbra.utils.render.shader.ShaderProgram;
import gg.umbra.wrapper.impl.DeltaTracker;

public class EventPreRenderTick
extends EventRenderTickBase {
    @Override
    public boolean fire() {
        ShaderProgram.setCurrentProgramId(-1);
        return super.fire();
    }

    public EventPreRenderTick(Object deltaTrackerHandle) {
        super(new DeltaTracker(deltaTrackerHandle));
    }

    public EventPreRenderTick(float f) {
        super(f);
    }
}
